math/factorial.cpp

```diff
@@ -1,20 +1,59 @@
 /**
  * @file
- * @brief C++ program to find factorial of given number
+ * @brief Find the [factorial](https://en.wikipedia.org/wiki/Factorial) of a given number
+ * @description calculates factorial via recursion
```

Comment:
AFAIK, `@description` is invalid. `@details` should be used.
```suggestion
 * @details Calculate factorial via recursion
```